### DISCUSSION TITLE
싱글 플레이 생성 질문

### DISCUSSION THREAD

**Discussion by zzoozzooo** [naverz/zepeto-studio-kor#731] [5. Q&A]
안녕하세요.

제페토로 월드 개발을 구상 중인데 싱글플레이 관련 내용을 찾지 못해서 질문 드립니다.

질문1.
멀티 플레이 모듈을 넣지 않으면 그냥 싱글플레이로 월드가 만들어지는 것인가요?
예시로 플레이어 A, B가 동시에 월드X에 접속을 해도
접속한 월드X에 혼자만 존재하도록 되는 것인지가 궁금합니다.

질문2.
멀티 플레이로 만들어서 로비와 같은 장소에서는 여러 플레이어들과 함께 있다가
오브젝트 등을 터치하면 싱글 씬으로 전환을 하여 혼자서만 월드를 플레이할 수 있도록 구현이 가능한가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 질문 주신 내용은 모두 말씀하신 대로 구현이 가능합니다.
멀티플레이 모듈을 넣지 않으면 싱글플레이이며 월드에 혼자만 존재합니다.
여러 씬을 구성해서 특정 씬만 멀티로 하는 것도 가능합니다.
감사합니다.

**Reply by zzoozzooo** [reply]
답변 감사합니다!!!  :)
